src/Card.tsx

```diff
@@ -284,95 +363,210 @@ export default function Card({
+                      icon={
+                        (!editMode && contact.star) || (editMode && draftStar)
+                          ? faStarSolid
+                          : faStarOutline
+                      }
+                    />
+                  </button>
+                )}
+              {editMode ? (
+                <div 
+                  className="rounded-full border-2 grid grid-rows-2 transition float-right"
+                  style={{
+                    background: THEMES[theme].color,
+                    borderColor: THEMES[theme].color,
+                  }}
+                >
+                  <button 
+                    className="p-1 px-2 rounded-full transition-all"
+                    style={draftCardType == "person" ? {
+                      background: THEMES[theme].background,
+                      color: THEMES[theme].color
+                    } : {
+                      background: "none",
+                      color: THEMES[theme].background
+                    }}
+                    onClick={() => setDraftCardType("person")}
+                  >
+                    <FontAwesomeIcon icon={faPerson}/>
+                  </button>
+                  <button 
+                    className="p-1 px-2 rounded-full transition-all"
+                    style={draftCardType == "repeater" ? {
+                      background: THEMES[theme].background,
+                      color: THEMES[theme].color
+                    } : {
+                      background: "none",
+                      color: THEMES[theme].background
+                    }}
+                    onClick={() => setDraftCardType("repeater")}
+                  >
+                    <FontAwesomeIcon icon={faTowerBroadcast}/>
+                  </button>
+                </div>
+              ) : (
+                contact.cardType == "person" ? (
+                  <FontAwesomeIcon icon={faPerson} className="text-4xl"/>
+                ) : (
+                  <FontAwesomeIcon icon={faTowerBroadcast} className="text-3xl"/>
+                )
+              )}        
+            </div>
+          </div>
+          {(editMode && draftCardType == "person") || (!editMode && contact.cardType == "person") ? (
+            <Input
+              className="text-3xl -my-1"
+              value={displayName}
+              onChange={(e) => setDraftName(e.target.value)}
+              onKeyDown={handleInputKeyDown}
+              placeholder="name"
+              disabled={!editMode}
+            />
+          ) : null} 
+          {(editMode && draftCardType == "repeater") || (!editMode && contact.cardType == "repeater") ? (
+            <Input
+              className="text-3xl -my1"
+              value={displayLocation}
+              onChange={(e) => setDraftLocation(e.target.value)}
+              onKeyDown={handleInputKeyDown}
+              placeholder="location"
+              disabled={!editMode}
+            />     
```

Comment:
these are now clipping into the buttons

<img width="372" alt="Screenshot 2024-07-26 at 20 36 15" src="https://github.com/user-attachments/assets/318a636b-5380-418d-b18e-e931b854027f">